modules/geoip2/maxminddb-helper.c

```diff
@@ -25,6 +25,11 @@
 #include <logmsg/logmsg.h>
 #include <messages.h>
 
+#ifndef __STDC_FORMAT_MACROS
+#define __STDC_FORMAT_MACROS
+#endif
+#include <inttypes.h>
+
```

Comment:
this one cries for me for a commonplace somewhere, instead of the repeated inclusion, but could not find the best one for it yet...

Reply:
ok, `lib/compat`` dir will do the trick

please create a `lib/compat/inttypes.h` or similar file

and include that one where needed